build-configs/common/theme/colors.ts

```diff
@@ -1,26 +1,26 @@
 export type ColorsType = {
   themeColor: string
-  themeColorLight: string
+  themeColorLight?: string
   backgroundAccentColor: string
   textColor: string
-  textSecondaryColor: string
-  textDecorationColor: string
-  textDisabledColor: string
-  footerLineColor: string
-  backgroundColor: string
-  tunewsThemeColor: string
-  tunewsThemeColorLight: string
+  textSecondaryColor?: string
+  textDecorationColor?: string
+  textDisabledColor?: string
+  footerLineColor?: string
+  backgroundColor?: string
```

Comment:
I made them optional, because of the error I was getting saying that those colors are absent

Reply:
I think a better option would be something like
``` ts
export const commonHighContrastColors = {
  ...commonLightColors,
  backgroundAccentColor: '#20293A',
  backgroundColor: '#101217',
  textColor: '#FFFFFF',
  borderColor: '#FFFFFF',
}
```

Reply:
For me this works if you adjust `build-configs/integreat/theme/colors.ts`as follows:
``` ts
export const highContrastColors: ColorsType = {
  themeColor,
  themeColorLight,
  themeContrast,
  ...commonHighContrastColors,
}
```